veerer/multiscale_veering_triangulation.py

```diff
@@ -925,88 +944,34 @@ def bdry_of_cyl(vt, r_low, h):
             l_horiz[abs(level)] = l1
         else:
             l_horiz.insert(abs(level) + 1, [])
-
+            l_horiz[abs(level)] = l1
+            l_horiz[abs(level) + 1] = l2
+        
         #build the vertical nodes
         l_pm = []
         #existing vertical nodes
         original_pm = copy.deepcopy(self._prong_matching)
         for pm in original_pm:
             prong1, prong2 = pm
-            if prong1[0] == (level, component):
+            if prong1[0] == (-abs(level), component):
                 prong1 = track_prong(vt, r_up, r_low, prong1)
                 if f_up is None:
                     l_pm.append([prong1,prong2])
                 else:
                     prong2 = ((prong2[0][0] - 1,prong2[0][1]), prong2[1], prong2[2])
                     l_pm.append([prong1,prong2])
-            elif prong2[0] == (level, component):
+            elif prong2[0] == (-abs(level), component):
                 prong2 = track_prong(vt, r_up, r_low, prong2)
                 l_pm.append([prong1,prong2])
             else:
                 l_pm.append(pm)
         #new vertical nodes
         if f_up is not None:
-            l_pm  = l_pm + _new_prong_matching(vt, f_low, r_up, r_low,level,component)
+            l_pm  = l_pm + _new_prong_matching(vt, f_low, r_up, r_low, -abs(level), component)
 
         #build the degeneration
         return MultiscaleVeeringTriangulation(vts,l_horiz,l_pm)
-
```

Comment:
@KaiFu2210 Why did you remove the two functions I introduced?

Reply:
that is `codimension_one_horizontal_degenerations` and `codimension_one_vertical_degenerations`?